src/components/PhotoGallery/PhotoGallery.tsx

```diff
@@ -1,6 +1,11 @@
 import axios from "axios";
 import { useCallback, useEffect, useState } from "react";
-import { PicturesContainer, Image } from "./PhotoGallery.styles";
+import { PicturesContainer,PicturesHeader } from "./PhotoGallery.styles";
```

Comment:
Prettier?

Reply:
You will need to install the plugin and fix formatting on the files you changed.

Reply:
done. Ben had helped me to install it. It just doesn't run automatically, so I had to do it myself